3_visualize/src/plot_model_data.r

```diff
@@ -1,9 +1,7 @@
-plot_data <- function() {
-  # Set file paths for data
-  project_input_dir <- '2_process/out'
-  project_output_dir <- '3_visualize/out'
+plot_data <- function(project_output_dir, data) {
+  
   # Create a plot
-  png(file = file.path(project_output_dir, 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
+  png(file = file.path(project_output_dir), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
No need to use the `file.path()` function here, since you're already passing in a complete filepath for the output file